lib/jsdom/browser/Window.js

```diff
@@ -193,6 +202,10 @@ function Window(options) {
           return this.responseText;
         } else if (this.responseType === "json") {
           return JSON.parse(this.responseText);
+        } else if (window.document.location.protocol === "file:") {
+          // node-xhr doesn't properly set the responseType for file requests.
+          // return the text anyway.
```

Comment:
Got it. Needs a bit of clarification in the comments, but I can fix that while merging in.